inputreader_windows.go

```diff
@@ -57,7 +57,7 @@ func newInputReader(r io.Reader) (cancelreader.CancelReader, error) {
 func (r *conInputReader) Cancel() bool {
 	r.setCanceled()
 
-	return windows.CancelIo(r.conin) == nil
+	return windows.CancelIoEx(r.conin, nil) == nil || windows.CancelIo(r.conin) == nil
```

Comment:
Do we need `windows.CancelIo` anymore?
```suggestion
	return windows.CancelIoEx(r.conin, nil) == nil
```

Reply:
I left the `windows.CancelIo` call in for the very slim chance that the call and the target input handler are on the same thread. I don't seem any harm in keeping it in the code, especially if the `windows.CancelIoEx` call addresses 99% of the use cases. That said, I'll leave the final decision up to you.